src/main/java/com/study_buddy/study_buddy/model/GroupMemberID.java

```diff
@@ -0,0 +1,45 @@
+package com.study_buddy.study_buddy.model;
+
+import jakarta.persistence.Embeddable;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Embeddable
+public class GroupMemberID implements Serializable {
+    private Long memberId;
+    private Long groupId;
```

Comment:
Add proper annotations

Reply:
See https://github.com/savipauk/study-buddy/blob/e14b057340740dfb4c4da6b7c900070814cba0c5/src/main/java/com/study_buddy/study_buddy/model/StudyGroup.java#L13 for example